components/cdc/src/delegate.rs

```diff
@@ -270,41 +281,49 @@ impl Delegate {
             for downstream in pending.downstreams {
                 self.subscribe(downstream);
             }
-            for (downstream_id, entries) in pending.scan {
-                self.on_scan(downstream_id, entries);
-            }
             for batch in pending.multi_batch {
                 self.on_batch(batch)?;
             }
```

Comment:
Should we check `observe_id`?

Reply:
We will check it inside the `on_batch` function.